Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -1043,4 +1044,35 @@ std::set<std::string> HoI4::World::getSouthAsianCountries() const
 	}
 
 	return southAsianCountries;
+}
+
+void HoI4::World::buildConquerStrategies()
+{
+	for (auto& country: countries)
+	{
+		for (const auto& strategy: country.second->getAIStrategies())
+		{
+			if (strategy.getType() == "conquer_prov")
+			{
+				if (auto HoI4Tag =
+						  countryMap.getHoI4Tag((*sourceWorld->getProvince(std::stoi(strategy.getID())))->getOwner());
+					 HoI4Tag)
+				{
+					const auto& conquerStrategies = country.second->getConquerStrategies();
```

Comment:
If any of this can be placed in the country file, it should be.

Reply:
Can I get Vic2 province ID from HoI4 country? That is, a province owned by another country.

Reply:
Not easily, I think. But this section doesn't need that, right? It's all things already internal to the country or things you've already calculated.

Reply:
The province ID gets transferred along with a "conquer_prov" Vic2 strategy containing it. Though I think I'll be able to process it in Vic2 country instead. Circumventing the stoi issue as well.